src/csharp/Grpc.Reflection/Grpc.Reflection.csproj

```diff
@@ -4,16 +4,18 @@
   <Import Project="..\Grpc.Core\Common.csproj.include" />
 
   <PropertyGroup>
-    <Copyright>Copyright 2016, Google Inc.</Copyright>
-    <AssemblyTitle>gRPC C# Reflection</AssemblyTitle>
+    <Authors>The gRPC Authors</Authors>
+    <Copyright>Copyright 2016 The gRPC Authors</Copyright>
```

Comment:
nit to update the copyright year here and elsewhere

Reply:
I think we should not update for pre-existing files?